ruby/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'tzinfo'
```

Comment:
Ugh, I misread your comment.  tzinfo-data is not referenced in `tzdump.rb`.
I'll fix that...